### PR TITLE
Animation

### PR DESCRIPTION
**Pull request by LordMittens**
Ik heb animaties toe gevoegd. player prefeb heeft nu een animator, er is een animation folder en Move Velocity is aangepast

### CODE REVIEW

**Comment by Dennisbonke** [comment]
Please use proper commit messages conform the commit style (see [here](https://docs.opa.rip)) and we use English on GitHub, so please switch to English.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 namespace OperationBlackwell.Player {
 	public class MoveVelocity : MonoBehaviour, IMoveVelocity {
 		[SerializeField] private float moveSpeed_;
+		[SerializeField] Animator animator;
```
Comment (Dennisbonke) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:7]: Missing modifier, lacking code style.

Comment (Westlanderz) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:7]: This misses the modifier `private` and a trailing underscore.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs [hunk 2]
```diff
@@ -20,6 +21,8 @@ public class MoveVelocity : MonoBehaviour, IMoveVelocity {
 
 		private void FixedUpdate() {
 			rigidbody2D_.velocity = velocityVector_ * moveSpeed_;
+			if (this.name == "Player")
```
Comment (Dennisbonke) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:24]: No spaces between `if` and `(`.

Comment (Dennisbonke) [reply Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:24]: And no `{` and `}` either, please check the code style.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs [hunk 3]
```diff
@@ -30,5 +33,18 @@ public class MoveVelocity : MonoBehaviour, IMoveVelocity {
 		public void Enable() {
 			this.enabled = true;
 		}
+		
+		private void updateAnimation()
```
Comment (Dennisbonke) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:37]: This entire function is scuffed, check the code style and reformat.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs (hunk 2 above)
Comment (Westlanderz) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:24]: Please make name a variable that can be set.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs [hunk 4]
```diff
@@ -30,5 +33,18 @@ public class MoveVelocity : MonoBehaviour, IMoveVelocity {
...
 		}
+		
+		private void updateAnimation()
+        {
+			Debug.Log(velocityVector_);
+
+			animator.SetFloat("Horizontal", velocityVector_.x);
+			animator.SetFloat("Vertical", velocityVector_.y);
+
+			if ((velocityVector_.x == 0) && (velocityVector_.y == 0))
```
Comment (Dennisbonke) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:44]: Doesn't match code style.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs [hunk 5]
```diff
@@ -30,5 +33,18 @@ public class MoveVelocity : MonoBehaviour, IMoveVelocity {
 		public void Enable() {
 			this.enabled = true;
 		}
+		
+		private void updateAnimation()
+        {
```
Comment (Westlanderz) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:38]: This is not correct with the code style.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs [hunk 6]
```diff
@@ -30,5 +33,18 @@ public class MoveVelocity : MonoBehaviour, IMoveVelocity {
...
+		private void updateAnimation()
+        {
+			Debug.Log(velocityVector_);
+
+			animator.SetFloat("Horizontal", velocityVector_.x);
+			animator.SetFloat("Vertical", velocityVector_.y);
+
+			if ((velocityVector_.x == 0) && (velocityVector_.y == 0))
+				animator.SetBool("isWalking", false);
+			else
```
Comment (Dennisbonke) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:46]: Doesn't match code style.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs (hunk 2 above)
Comment (Westlanderz) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:24]: Code style issues.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Styling errors, please fix. Also, English is used here, so translate your PR message to English and write proper commit messages.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs [hunk 7]
```diff
@@ -30,5 +33,18 @@ public class MoveVelocity : MonoBehaviour, IMoveVelocity {
 		public void Enable() {
 			this.enabled = true;
 		}
+		
+		private void updateAnimation()
+        {
+			Debug.Log(velocityVector_);
```
Comment (Westlanderz) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:39]: Remove unnecessary `Debug.Logs`.

File: Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs (hunk 4 above)
Comment (Westlanderz) [inline Assets/Scripts/Player/MoveVelocity/MoveVelocity.cs:44]: More code style issues.

**Review by Westlanderz — CHANGES_REQUESTED** [review]

**Review by Westlanderz — APPROVED** [review]

**Review by Westlanderz — DISMISSED** [review]

**Review by Dennisbonke — APPROVED** [review]
LGTM now.

**Review by Westlanderz — APPROVED** [review]
LGTM

**Pull request merged**